src/main/java/maksym/fedorenko/bookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package maksym.fedorenko.bookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl"
+)
+public class MapperConfig {
```

Comment:
this class makes no sense :)

it is better to specify @MapperConfig directly on the mapper class